ROADMAP.md

```
@@ -0,0 +1,24 @@
+# Implementation Roadmap
+
+Items migrated from the strategy repo pipeline (March 2026). These are implementation-level tasks that belong in this repo.
+
+## Active
+
+### Code4Compassion — Teacher & Admin
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)